### DISCUSSION TITLE
Context propagation in OpenTelemetry tracer

### DISCUSSION THREAD

**Discussion by mitoeth** [open-telemetry/opentelemetry-dotnet#3204] [Q&A] [answered]
Hello everyone,

New to OpenTelemetry. I would like to know what context propagation headers are used by default when instrumenting the application at startup - or is there a way to confirm this. I know OpenTelemetry supports B3 and W3C. Looking through the repository, I believe I can see code for both use cases if I'm not mistaken. Does the context propagation depend on the exporter? I use Zipkin

Thanks to everyone contributing to this project.

**Comment by cijothomas** [comment, marked as answer]
W3CTraceContext and W3CBaggage propagators are enabled by default.
https://github.com/open-telemetry/opentelemetry-dotnet/tree/main/src/OpenTelemetry.Api#context-propagation this section (could use some improvements..)

**Reply by mitoeth** [reply]
Thanks for the help

**Comment by cijothomas** [comment]
> Does the context propagation depend on the exporter?

No
